### PR TITLE
[new release] netkat (0.1)

### PR DESCRIPTION
**Pull request by cakoch10**
A clean slate implementation of NetKAT

- Project page: <a href="https://github.com/netkat-lang/netkat">https://github.com/netkat-lang/netkat</a>
- Documentation: <a href="https://netkat-lang.github.io/netkat/">https://netkat-lang.github.io/netkat/</a>

##### CHANGES:

- Initial release

### CODE REVIEW

**Comment by mseri** [comment]
Looks like the failure is due to nice_parser

```
Processing 83/83:
[ERROR] The sources of the following couldn't be obtained, aborting:
          - nice_parser.1.0.0:
            https://github.com/smolkaj/ocaml-parsing/archive/1.0.0.tar.gz (Bad checksum, expected md5=9ac52ebf61b297e0eed7821e714efa23)
```

@smolkaj do you know what happened? Has the package changed or it is just Github recompressing the artifacts?

File: packages/netkat/netkat.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+]
+bug-reports: "https://github.com/netkat-lang/netkat/issues"
+homepage: "https://github.com/netkat-lang/netkat"
+doc: "https://netkat-lang.github.io/netkat/"
+license: "MIT"
+dev-repo: "git+https://github.com/netkat-lang/netkat.git"
+synopsis: "A clean slate implementation of NetKAT"
+description: "A clean slate implementation of NetKAT"
+depends: [
+  "async" {>= "0.12.0"}
```
Comment (kit-ty-kate) [inline packages/netkat/netkat.0.1/opam]:
```suggestion
  "async" {>= "v0.12.0"}
```

**Comment by kit-ty-kate** [comment]
ping @cakoch10. The checksum is still failing

**Comment by kit-ty-kate** [comment]
CI restarted.

**Comment by kit-ty-kate** [comment]
thanks! sorry for the delay

**Pull request merged**